client/components/common/tce-core/utils.js

```diff
@@ -32,6 +32,12 @@ export function getElementId(element) {
   return element && (element.uid || element.id);
 }
 
+export const getCommentEvents = () => ({
```

Comment:
Please replace with standard enum like const. Instead of utils create Events dir and expose CommentEvent/DiscussionEvent